<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SkillPackEditor</class>
 <widget class="QMainWindow" name="SkillPackEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">
QPushButton:pressed {color:red}

QWidget { border: 1px solid lightgray}

QLabel {border: 1px solid gray}
QPushButton {border: 1px solid gray}

QPushButton:hover { border-radius:5px; background:lightgreen}
QLabel:hover { border-radius:5px; background:lightgreen}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="skillPackTree">
      <item>
       <widget class="QLabel" name="skillpakcTreeLabel">
        <property name="text">
         <string>Skillpack tree:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeView" name="tree"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="skillpackeditorbuttons">
        <item alignment="Qt::AlignTop">
         <widget class="QPushButton" name="AddSkill">
          <property name="text">
           <string>Add skill</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="AddLevel">
          <property name="text">
           <string>Add level</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="remove">
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="toolsAndDescriptionWidget_2" native="true">
      <layout class="QVBoxLayout" name="toolsAndDescriptionWidget">
       <item>
        <widget class="QLabel" name="SettingsLabel">
         <property name="text">
          <string>Skill pack settings and information:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="tools" native="true">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="currentFileLabel">
            <property name="text">
             <string>Current file:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="currentFileName">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Total skills:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="totalSkills">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="skillpackNAmeLabel">
            <property name="text">
             <string>Name:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="skillPackName">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="autoSave">
         <property name="text">
          <string>Autosave</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionCreate"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionClose"/>
    <addaction name="actionReturn_to_launcher"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp_me"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Dev</string>
    </property>
    <addaction name="actionSet_style"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
   <addaction name="menuDebug"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCreate"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionClose"/>
   <addaction name="separator"/>
   <addaction name="actionHelp_me"/>
   <addaction name="separator"/>
   <addaction name="actionReturn_to_launcher"/>
   <addaction name="actionSet_style"/>
  </widget>
  <action name="actionCreate">
   <property name="icon">
    <iconset resource="../Resources/Resources.qrc">
     <normaloff>:/icons/Icons/CreateSkillPackFile.png</normaloff>:/icons/Icons/CreateSkillPackFile.png</iconset>
   </property>
   <property name="text">
    <string>Create skillpack file</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../Resources/Resources.qrc">
     <normaloff>:/icons/Icons/OpenSkillPackFile.png</normaloff>:/icons/Icons/OpenSkillPackFile.png</iconset>
   </property>
   <property name="text">
    <string>Open skillpack file</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../Resources/Resources.qrc">
     <normaloff>:/icons/Icons/SaveSkillPackFile.png</normaloff>:/icons/Icons/SaveSkillPackFile.png</iconset>
   </property>
   <property name="text">
    <string>Save skillpack file</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="../Resources/Resources.qrc">
     <normaloff>:/icons/Icons/CloseSkillPackFile.png</normaloff>:/icons/Icons/CloseSkillPackFile.png</iconset>
   </property>
   <property name="text">
    <string>Close skilpack file</string>
   </property>
  </action>
  <action name="actionHelp_me">
   <property name="icon">
    <iconset resource="../Resources/Resources.qrc">
     <normaloff>:/icons/Icons/Help.png</normaloff>:/icons/Icons/Help.png</iconset>
   </property>
   <property name="text">
    <string>Help me!</string>
   </property>
  </action>
  <action name="actionReturn_to_launcher">
   <property name="icon">
    <iconset resource="../Resources/Resources.qrc">
     <normaloff>:/icons/Icons/ReturnToLauncher.png</normaloff>:/icons/Icons/ReturnToLauncher.png</iconset>
   </property>
   <property name="text">
    <string>Return to launcher</string>
   </property>
  </action>
  <action name="actionSet_style">
   <property name="icon">
    <iconset resource="../Resources/Resources.qrc">
     <normaloff>:/icons/Icons/SetStyle.png</normaloff>:/icons/Icons/SetStyle.png</iconset>
   </property>
   <property name="text">
    <string>Set style (Developers only)</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../Resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
